CHANGELOG.md

```diff
@@ -1,53 +1,64 @@
 # Mattermost Desktop Application Changelog
 
-## UNDER DEVELOPMENT
+## Release v3.7.0
 
-The "UNDER DEVELOPMENT" section of the Mattermost Desktop changelog appears
-in the product's `master` branch to note key changes committed to `master`
-and are on their way to the next stable release.
-When a stable release is pushed, "UNDER DEVELOPMENT" heading is removed
-from the final changelog of the release.
-
-Release date: TBD
+Release date: May 9, 2017
 
 ### Improvements
-- Added spell checker for English, French, German, Spanish, and Dutch.
-- Removed "Display secure content only" option it's no longer necessary.
-- Reset the window position if it's restored off-screen from minimized state.
+
+#### All Platforms
+ - Added an inline spell checker for English, French, German, Spanish, and Dutch. [#225](https://github.com/mattermost/desktop/issues/225)
+ - Removed an obsolete "Display secure content only" option, following an [upgrade of the Electron app to Chrome v56](https://github.com/electron/electron/commit/2e0780308c7ef2258422efd34c968091d7cd5b65). [#469](https://github.com/mattermost/desktop/pull/469)
+ - Refactored main app window code for ... [#470](https://github.com/mattermost/desktop/pull/470)
```

Comment:
I'm not sure if there's anything we can mention here that the PR improved (e.g. performance, rendering, .... ?)